src/screens/SponsorScreen/__snapshots__/component.test.js.snap

```diff
@@ -173,8 +173,8 @@ exports[`SponsorScreen Component renders correctly 1`] = `
           Object {
             "text": Object {
               "color": "#2d2f7f",
-              "fontSize": 14,
-              "lineHeight": 20,
+              "fontSize": 9,
+              "lineHeight": 13,
```

Comment:
Intentional? I think the change is right, but not 100%.

Also, I was thinking about this. Should we just make a component to do this rather than going through markdown? This is a custom look (small purple text with bullets) and I think we might be better off just creating an UnorderedList and ListItem component instead.

Reply:
Intentional, yes. And I think you are right about the list. A custom list component would be simpler.